fili-core/src/main/java/com/yahoo/bard/webservice/data/config/dimension/LookupDefaultDimensionConfig.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config.dimension;
+
+import com.yahoo.bard.webservice.data.config.names.DimensionName;
+import com.yahoo.bard.webservice.data.dimension.DimensionField;
+import com.yahoo.bard.webservice.data.dimension.KeyValueStore;
+import com.yahoo.bard.webservice.data.dimension.SearchProvider;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+/**
+ * A Lookup Dimension instance holds all of the information needed to construct a Lookup Dimension.
+ */
+public class LookupDefaultDimensionConfig extends DefaultDimensionConfig implements LookupDimensionConfig {
```

Comment:
I do not see the need to have an default implementation for every implementation of `DimensionConfig` since most likely users needs to extend or implement something in fili application anyways. So simply extending `BaseDimensionConfig` would do the job I think. Convince me otherwise.